src/renderer/features/network/CustomRpcForm/model/__test__/edit-custom-rpc-model.test.ts

```diff
@@ -0,0 +1,110 @@
+import { fork, allSettled } from 'effector';
+
+import { editCustomRpcModel } from '../edit-custom-rpc-model';
+import { ConnectionStatus, RpcNode } from '@shared/core';
+import { ExtendedChain } from '@entities/network';
+import { RpcCheckResult } from '../../lib/types';
+
+describe('features/network/CustomRpcForm/edit-custom-rpc-model', () => {
+  test('should have the form pre-filled on launch', async () => {
+    const scope = fork({});
+    const mockNode = { name: 'some name', url: 'some url' } as RpcNode;
+
+    await allSettled(editCustomRpcModel.events.nodeSelected, { scope, params: mockNode });
+
+    expect(scope.getState(editCustomRpcModel.$editCustomRpcForm.$values)).toEqual(mockNode);
+  });
+
+  test('should have error for wrong node url', async () => {
```

Comment:
The thing is that it's not because the rule is set in the form, that the UI will correctly implement and show the error. To me the model can be perfectly implemented, but the UI could miss the displaying of the error, which is a pretty annoying UX issue, as in users won't know what's up.

Reply:
There was a misunderstanding there, I wasn't finished with tests actually. This is fixed now